kadena-signing-api/default.nix

```diff
@@ -1,14 +1,8 @@
-{ pactRef ? "ace18a2489bf86595af09316bd1c09794c326b77"
-, pactSha ? "13cg8nz8lc839xnfp31yj8ihra5qpgfz9g4lzqmgcfk4kmxcvlxi"
-, kpkgs ? import ./dep/kpkgs {}
+{ kpkgs ? import ./dep/kpkgs {}
 }:
 
 let
-pactSrc = builtins.fetchTarball {
-  url = "https://github.com/kadena-io/pact/archive/${pactRef}.tar.gz";
-  sha256 = pactSha;
-};
-
+pactSrc = /Users/doug/work/pact;
```

Comment:
It won't let me push the fix myself, so im adding this here just to make sure no one forgets: 
```
  pactSrc = kpkgs.pkgs.fetchFromGitHub {
    owner = "kadena-io";
    repo = "pact";
    rev = "4971ab6078b75eb612d83d56f1e7cd139a5a2ba8";
    sha256 = "1jx1hd596r5rrx96r2v2xds6pjjmi4lfk7xm14f3gkx2gmavgyr3";
  };
```